src/HealthKit/Enums.cs

```diff
@@ -534,15 +534,24 @@ public enum HKSwimmingStrokeStyle : long {
 	[Watch (4, 0), iOS (11, 0)]
 	[Native]
 	public enum HKInsulinDeliveryReason : long {
+		[Obsolete ("Use 'Basal' instead.")]
```

Comment:
We really need to make sure we do not ship this in the future, you need to ad a #if XAMCORE_4_0 check here